app/views/groups/show.html.erb

```diff
@@ -10,3 +10,30 @@
   <%= t('items.headlines.items') %>
 </h5>
 <%= render GroupItemListComponent.new(group: @group) %>
+<table class="table table-bordered">
```

Comment:
The member count should also be displayed as seen in the wireframe.
Also already done in [edit.html.erb](https://github.com/hpi-swt2/bookkeeper-portal-red/blob/dev/app/views/groups/edit.html.erb#L10-L12)